.grit/patterns/js/no_implicit_coercion.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Disallow shorthand type conversions
+tags: [good-practice]
+---
+This matches the [eslint rule](https://eslint.org/docs/latest/rules/no-implicit-coercion).
+
+Check the [Playground](https://app.grit.io/studio?key=cle6qCCw_-XMW9Nxs-PKH)
```

Comment:
Don't include playground links in the library, since it automatically gets inserted anyways.